target/linux/brcm63xx/Makefile

```diff
@@ -11,7 +12,7 @@ BOARD:=brcm63xx
 BOARDNAME:=Broadcom BCM63xx
 SUBTARGETS:=generic smp
 FEATURES:=squashfs usb atm pci pcmcia usbgadget
-KERNEL_PATCHVER:=4.1
+KERNEL_PATCHVER:=4.4
 MAINTAINER:=Jonas Gorski <jogo@openwrt.org>
 
 include $(INCLUDE_DIR)/target.mk
```

Comment:
(because github doesn't allow commenting on the commit log):

You are doing three things here:
a) switch to 4.4
b) drop 4.1 support
c) refresh config-4.4

Please split it accordingly, and add commit logs (well, no need for the refresh one). The switch to 4.4 should say if you tested it.